modules/ROOT/pages/monitoring/logging.adoc

```diff
@@ -1,215 +1,334 @@
+In version 4.0, this was configured with the parameter `dbms.logs.user.rotation.size`.
 
-| 28, 29
-| Defines a logger that matches log events with the `HttpLogger` target with a log level of `INFO` or above.
-The `additivity="false"` is set to fully consume the log event.
+<8> The `DefaultRolloverStrategy` defines how many files to keep as history. +
+The `fileIndex=min` implies that the minimum/the lowest number is the most recent one. +
+The `max` attribute defines the number of files to keep as history before they are deleted, in this case, 7 files.
+In version 4.0, this was configured with the parameter `dbms.logs.user.rotation.keep_number`.
+
+<9> Loggers are used to define the log level and which appender to use for the log messages.
+The loggers are referenced by the `name` attribute.
+
+<10> The root logger is a "catch-all" logger that catches everything that is not caught by the other loggers and sends it to the appender(s) specified in the `AppenderRef` element(s).
+The root logger is referenced by the `Root` element.
+It can be set to `DEBUG`, `INFO`, `WARN`, `ERROR`, or `OFF`.
+The default log level is `INFO`.
+
+<11> You can also define custom loggers to catch specific log events and send them to the appender(s) specified in the `AppenderRef` element(s).
+For example, the `QueryLogger` logger (configured in _server-logs.xml_) is used to catch log events with a log level of `INFO` or above and send them to the `QueryLog` appender. +
+The `additivity="false"` is set to fully consume the log event and not send it to the root logger. +
 If `additivity="true"` is set, which is the default, the log event is also sent to the root logger.
-|===
+
+== Possible logging customizations
+
+The default logging configuration is a good starting point, but you might want to customize it to your needs.
+The following sections describe the Log4j configuration elements and how to use them to customize the logging configuration.
 
 [[general-logging-appenders]]
 === Appenders
-An appender represents a destination for log events.
 All Log4j standard appenders are available in Neo4j.
-For more details, see the https://logging.apache.org/log4j/2.x/manual/appenders.html[Log4j official documentation].
-A few of the most common appenders are `<Console>`, `<RollingFile>`, and `<RollingRandomAccessFile>`.
+For more details, see the https://logging.apache.org/log4j/2.x/manual/appenders.html[Log4j official documentation on appenders].
 
-==== `<Console>` appender
-The console appender outputs log events to _stdout_ or _stderr_.
+A few of the most common appenders are `<RollingRandomAccessFile>`, `<RollingFile>`, and `<Console>`.
 
-.An example of a console appender
-[source,xml]
-----
-<Console name="console" target="SYSTEM_OUT"> <!-- or SYSTEM_ERR -->
-  <PatternLayout pattern="%m%n"/>
-</Console>
-----
+==== `<RollingRandomAccessFile>` appender
+The `<RollingRandomAccessFile>` is the default appender in Neo4j.
```

Comment:
a lof of info in this section is a duplication from the previous section with numbers 1-11. Is that intentional?

Reply:
Yes, I did it because nobody reads the whole page. So in this way, the Appenders section is self-contained and compact if they happen to read just this section.